raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -347,3 +520,66 @@ def test_unsubscribe_from_peers(comms_clients):
     client_2.unsubscribe_from(client_1.rsk_address.address)
     assert client_2._is_subscribed_to(client_1.rsk_address.address) is False
     assert client_1._is_subscribed_to(client_2.rsk_address.address) is False  # check operations are independent
+
+
+def test_client_timeouts():
+    nodes = []
+    try:
+        node_1 = CommsNode(CommsConfig(node_id="A", amount_of_clients=1, auto_connect=False))
+        nodes.append(node_1)
+        client_1, address_1 = node_1.clients[0], node_1.clients[0].rsk_address.address
+
+        node_2 = CommsNode(CommsConfig(node_id="B", amount_of_clients=1, auto_connect=False))
+        nodes.append(node_2)
+        client_2, address_2 = node_2.clients[0], node_2.clients[0].rsk_address.address
+
+        tiny_timeout = 1e-100
+
+        client_1.grpc_client_timeout = tiny_timeout
```

Comment:
i would just put   `client_1.grpc_client_timeout = 1e-100` and remove the `tiny_timeout` var

Reply:
True will fix that